src/lib/box-styles.tsx

```diff
@@ -152,6 +152,26 @@ export const Playground: React.FC<{
   );
 };
 
+type StateType = {
+  change: (value: any) => any;
+  children: (value: any, change: any) => void;
+  initial: any;
+};
+
+export const State = ({ initial, change, children }: StateType) => {
+  const [value, setValue] = React.useState(false);
+
+  React.useEffect(() => {
+    setValue(initial);
+  }, [initial]);
```

Comment:
Не очень понял эту секцию.

```suggestion
  const [value, setValue] = React.useState(initial);
```

Мб я немного не понял intention, но зачем здесь useEffect, если initial можно сразу закинуть в useState ?